### ISSUE TITLE
TrufflyRuby uses a lot of memory and high cpu usage while running Rubocop specs

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1505]
When I run Rubocop specs, TruffleRuby behaves like the title says.

In my case, TruffleRuby `1.0.0-rc9` on macOS 10.4 uses almost 3 GB RAM and CPU is 200 % for the most time.

To reproduce
```
git clone https://github.com/rubocop-hq/rubocop.git
cd rubocop
bundle exec rake spec
```

![screenshot 2018-12-05 at 18 21 33](https://user-images.githubusercontent.com/33736/49531727-cd7a1f80-f8b2-11e8-9410-c473c3c9b853.png)

**Comment by MaxLap** [comment]
Rubocop uses the `parser` gem, which can use lots of memory. See #1493

**Comment by deepj** [comment]
True, true, so this seems to be a duplication of #1493

**Comment by deepj** [comment]
Closed in a favour of #1493.

@MaxLap thx, for letting me know
